UPDATING.md

```diff
@@ -24,6 +24,9 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Removal of Mesos Executor
+The Mesos Executor is removed from the code base as it was not widely used and not maintained. [Mailing List Discussion on deleting it](https://lists.apache.org/list.html?dev@airflow.apache.org:lte=1M:mesos).
```

Comment:
This isn't a permalink btw :)

Reply:
Raised a PR to fix it - https://github.com/apache/airflow/pull/5476 . Thanks :)